backend/config.js

```diff
@@ -1,3 +1,11 @@
+const phaseName = () => {
+  const nameFromEnv = process.env.SYSTEM_PHASE
+  if (nameFromEnv === 'DEV' || nameFromEnv === 'PRE-PRODUCTION') {
```

Comment:
Not sure we need to be this strict. I'd probably just return the name.  

From a javascript standpoint I'd might write the following as:

`
const validPhaseNames = ['DEV', 'PRE_PRODUCTION']
return null ?? validPhaseNames.includes(process.env.SYSTEM_PHASE)
`